sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -731,6 +746,18 @@ def translate_scalar_counter_float(accumulator, metric_update_proto):
     metric_update_proto.floatingPoint = accumulator.value
 
 
+class _ParameterizedStager(Stager):
```

Comment:
<!--new_thread; commit:5061362ff0473be448ab872c9c279aa498f93b5a; resolved:0-->
Let's call this `_LegacyDataflowStager`, and take an instance of `DataflowApplicationClient` in the constructor, then let's call `dataflow_application_client._gcs_file_copy()` in stage_artifact.